src/CognitoCustomMessageProcessor/CognitoCustomMessageProcessor.csproj

```diff
@@ -12,14 +12,14 @@
     <PackageReference Include="Amazon.Lambda.Logging.AspNetCore" Version="3.1.0" />
     <PackageReference Include="Amazon.Lambda.Serialization.SystemTextJson" Version="2.2.0" />
     <PackageReference Include="AWSSDK.Core" Version="3.7.6.4" />
-    <PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.1" />
+    <PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.7" />
     <PackageReference Include="contentful.csharp" Version="7.3.0" />
     <PackageReference Include="Dte.Common" Version="1.7.4" />
     <PackageReference Include="Dte.Common.Lambda" Version="1.1.3"/>
     <PackageReference Include="Handlebars.Net" Version="2.1.4" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0-preview.7.23375.6" />
```

Comment:
Do we really want to be upgrading to preview versions of .NET 8 packages? What about .NET 7?